examples/recommendations/get-budget-reco.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This example is to get budget recommendations for Performance Max; 
+
+The response includes weekly impact metrics for the recommended budgets.
+
+Example uses: 
```

Comment:
At first I read this as "example uses" like "example use cases". Can you use a complete sentence to make it more clear?

Also can you remove the period after "strategy" for consistency?